yarn-project/aztec/src/testing/aztec_cheat_codes.ts

```diff
@@ -2,18 +2,18 @@ import { Fr } from '@aztec/foundation/fields';
 import { createLogger } from '@aztec/foundation/log';
 import type { AztecAddress } from '@aztec/stdlib/aztec-address';
 import { deriveStorageSlotInMap } from '@aztec/stdlib/hash';
-import type { AztecNode, PXE } from '@aztec/stdlib/interfaces/client';
-import type { Note } from '@aztec/stdlib/note';
+import type { AztecNode } from '@aztec/stdlib/interfaces/client';
+import type { Note, NotesFilter, UniqueNote } from '@aztec/stdlib/note';
 
 /**
  * A class that provides utility functions for interacting with the aztec chain.
  */
 export class AztecCheatCodes {
   constructor(
     /**
-     * The PXE Service to use for interacting with the chain
+     * The test wallet or pxe to use for getting notes
      */
-    public pxe: PXE,
+    public testWalletOrPxe: { getNotes(filter: NotesFilter): Promise<UniqueNote[]> },
```

Comment:
Using TestWallet here would make this unusable for externals as it's likely their wallet will not expose the getNotes method. Just having the method defined in the type makes this PR a non-breaking change. Everything else should not be facing externals.

Reply:
If this is unused, I would drop the `loadPrivate` method. That is a (testing) wallet utility